outsite.nl/components/pages/barbuddy/BarbuddyForm.vue

```diff
@@ -12,19 +12,23 @@ const props = defineProps({
   barBuddies: { type: Array, required: true },
 })
 
-const form = useReMemberForm('barbuddy', {
+const form = useReMemberForm('barbuddy2', {
```

Comment:
hm, I forgot, why can't we use the old form still?

Since we're just comma seperating the bar buddies I think we could just as well stick them into the original `barbuddy` property